xmtp-inbox-ios/Loaders/MessageLoader.swift

```diff
@@ -14,29 +14,35 @@ class MessageLoader: ObservableObject {
 	var client: XMTP.Client
 	var conversation: DB.Conversation
 
+	let fetchLimit = 10
+
+	@MainActor @Published var mostRecentMessageID = ""
 	@MainActor @Published var messages: [DB.Message] = []
 
 	init(client: XMTP.Client, conversation: DB.Conversation) {
 		self.client = client
 		self.conversation = conversation
 
+		do {
+			try fetchLocal()
```

Comment:
Moving this to init means that the messages are already there when you get to the convo detail view. In a follow up it'd probably be good to paginate locally from the DB as well.

Reply:
Makes sense to me! Although heads up that I did pull this branch down and am seeing trouble with an async function being called here:

<img width="761" alt="Screenshot 2023-02-14 at 9 01 33 AM" src="https://user-images.githubusercontent.com/556051/218760304-691bb253-3d98-4ddf-9db5-9d99cb6e7afd.png">

Reply:
Ooops, Ya I think that was the result of a bad merge.